batch/vep/Dockerfile

```diff
@@ -58,4 +58,6 @@ RUN perl INSTALL.pl \
 
 ADD run_vep.sh ${VEP_BASE}/run_vep.sh
 
+ADD run_vep_in_variant_transforms.sh ${VEP_BASE}/run_vep_in_variant_transforms.sh
```

Comment:
Lets have a more generic name as variant transform is too specific, and there is nothing from VT here. How about run_vep_with_watchdog.sh?

Actually it would be cleaner to have a watchdog script which accepts name of a script to run (in this case VEP) and a file to monitor. Watchdog scrip would be just a wrapper that runs the given script and kills it if the file is not updated for a certain time. This way we don't mix the two logic in a single file. Also DRY.

Reply:
Thank you Nima! I like the idea to separate the two logic! But how about the environment variables? The run_vep.sh still needs some environment variables, do I have to handle it in this script?